src/index.js

```diff
@@ -451,9 +439,6 @@ class Knob extends Component<KnobProps, KnobState> {
             {skinElemUpdates}
           </SvgLoader>
         </div>
-        {this.state.svgRef && (
-          <RotateView svg={this.state.svgRef} angle={angle} />
```

Comment:
I have removed `<RotateView>` here because I think this component is a complete no-op and we could delete the file. This is the only place in the project where it is used.

Reply:
update - I have removed it.